x-pack/plugins/monitoring/dev_docs/reference/indices.md

```diff
@@ -0,0 +1,40 @@
+This document provides a general overview of the indices used by the Stack Monitoring UI.
+
+## Index Patterns
+
+Stack Monitoring indices will generally fall into to `.monitoring-*` index pattern with some variation depending on collection mode.
+
+When using [internal collection](data_collection_modes.md#internal-collection), or standalone metricbeat collection prior to 8.0, data will be stored in dated raw indices following the pattern:
+
+- `.monitoring-(component)-7-(date)`
+
+When using [standalone metricbeat collection](data_collection_modes.md#standalone-metricbeat-collection) after 8.0, data will be stored in a data stream with the following pattern:
+
+- `.monitoring-(component)-8-mb`
+
+> Note that the distinction between 7 and 8 is the **data schema** version. These numbers are not expected to track stack release versions.
```

Comment:
Makes perfect sense, it's obvious really. (not)

Is there anything we can do to remove that confusion?

Reply:
I think the _best_ solution would be an elasticsearch level schema migration support like I mentioned in (internal) https://docs.google.com/document/d/11nnuLak7G6H88C1JQK08g8znjeyuhMrdXUNFziuDMRA/edit

But a short term option could be artificially bumping the schema version (to 11 of course 😉 ) in the next minor release.